docs/admin_api/rooms.md

```diff
@@ -391,13 +391,16 @@ The new room will be created with the user specified by the `new_room_user_id` p
 as room administrator and will contain a message explaining what happened. Users invited
 to the new room will have power level `-10` by default, and thus be unable to speak.
 
-If `block` is `True` it prevents new joins to the old room.
+If `block` is `true`, users will be prevented from joining the old room.
+This option can also be used to pre-emptively block a room, even if it's unknown
+to this homeserver. (If `block` is `false`, attempting to delete an unknown room
+is invalid and will be rejected as a bad request.)
```

Comment:
```suggestion
If `block` is `true`, users will be prevented from joining the old room.
This option can also be used to pre-emptively block a room, even if it's unknown
to this homeserver. In this case, the room will be blocked, and no further action
will be taken. If `block` is `false`, attempting to delete an unknown room is
invalid and will be rejected as a bad request.
```

Final bit of clarification after reading through it.

Reply:
Thanks. I'll update the corresponding docstring too.